docker/unsonic-init

```diff
@@ -1,16 +1,36 @@
 #!/bin/bash
 set -e
 
-source /unsonic/bin/activate
+# default username to 'admin', randomly generated password
+UNSONIC_USER="${UNSONIC_USER:-admin}"
+UNSONIC_PASSWORD="${UNSONIC_PASSWORD:-$(pwgen 12)}"
+
+unsonic_set_password() (
+    set -eu -o pipefail
+    local user="$1"; shift
+    local pass="$1"; shift
+
+    exec 1>&2
+    $UNSONIC password "$user" "$pass"
+)
 
 UNSONIC="unsonic -c /unsonic/etc/config.ini"
```

Comment:
Perhaps /unsonic/etc/config.ini should be added to the unsonic/config.DEF_LOCS for searching for the config file.

Reply:
Yah. I was trying not to complete redo it, but I think it'd make sense in the Docker world to go ahead and land the DB + config file in the same directory for the `VOLUME` mount. Something like:

1. embed a "default" `config.ini` in the image
2. have `config.ini` and the database live in the `VOLUME` (documented in README / on Docker page)
3. when user starts up container, the default config will be copied onto the VOLUME if it doesn't exist (could live under on the VOLUME in the image)
4. ???
5. Profit.

I might go ahead and try implementing this to see how it comes together.